WalletWasabi/Extensions/ProcessExtensions.cs

```diff
@@ -13,7 +13,7 @@ public static async Task WaitForExitAsync(this Process process, CancellationToke
 		{
 			while (!process.HasExited)
 			{
-				await Task.Delay(100, cancellationToken);
+				await Task.Delay(100, cancellationToken).ConfigureAwait(false); // TODO: https://github.com/zkSNACKs/WalletWasabi/issues/1452
```

Comment:
This could be in a different PR because it seems to be part of a different work.

Reply:
No, it cannot. The asyncmutex is in this PR and the asyncmutex's MutexThread is what hangs on after UI shutdown. The mutex doesn't doesn't get disposed, because for some strange reason the UI thread doesn't call properly dispose on using and cease existing before the await would have a chance to switch back to the UI thread context.